modules/ROOT/pages/dashboards/getting-started.adoc

```diff
@@ -0,0 +1,99 @@
+= Getting started with dashboards
+:description: Follow these steps for a working example of Neo4j Aura dashboards.
+
+Set up a working example of Neo4j Aura dashboards.
+
+== Prerequisites
+
+You need:
+
+. A Neo4j Aura account
+. A Neo4j Aura database instance
+
+See the Aura xref::/getting-started/quick-start-guide.adoc[Quick start guide] for details.
+
+== Add a sample data set
+
+Download the Neo4j link:https://github.com/neo4j-graph-examples/recommendations/raw/refs/heads/main/data/recommendations-50.dump[movie recommendations sample data set dump file].
+See link:https://neo4j.com/docs/getting-started/appendix/example-data/#_available_datasets[Available datasets] for alternative datasets.
+However, the rest of this page refers to the movie recommendations dataset.
+
+To add the sample data set in Aura:
```

Comment:
The way we make examples datasets available for people will change with the new PLG initiative, it will likely just be a button from the developer home. I'm wondering whether we should write the docs to use the new easy samples when available.